v2/support/basic_ticket_sample_app/assets/iframe.html

```diff
@@ -0,0 +1,44 @@
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+  <section data-main>Loading...</section>
+  <!-- https://github.com/zendesk/zendesk_app_framework_sdk -->
+  <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
+  <script>
+    // Initialise the Zendesk JavaScript API client
+    // https://developer.zendesk.com/apps/docs/agent/zaf_v2
+    var client = ZAFClient.init();
+
+    function renderMain(html) {
+      var mainSectionEl = document.querySelector('section[data-main]');
+      mainSectionEl.innerHTML = html;
+    }
+
+    function getCurrentUser() {
+      return client.get('currentUser').then(function(data) {
+        return data['currentUser'];
+      });
+    }
+
+    function init() {
+      getCurrentUser().then(function(currentUser) {
+        renderMain('<p>Hi <strong>' + currentUser.name + '</strong>, please make some changes to the ticket.');
+
+        client.on('*.changed', function(evt) {
+          var html = '<p><strong>' + currentUser.name + '</strong> changed <em>' +
+            evt.propertyName + '</em> to <em>' + evt.newValue + '</em>.</p>';
```

Comment:
technically an XSS vector... not sure if it's ok for a demo or if we should demonstrate best practices since developers will be learning off this code.